docs/content/getting-started/navigating-the-viewer.md

```diff
@@ -3,21 +3,21 @@ title: Navigating the viewer
 order: 500
 ---
 
-This guide will familiarize you with the basics of using the Rerun Viewer with an example dataset. By the end you should
-be comfortable with the following topics:
+This guide will familiarize you with the basics of using the Rerun Viewer with an example dataset. By the end you should be comfortable with the following topics:
 
-* [Launching the demo](#launching-the-demo)
-* [The Viewer panels](#the-viewer-panels)
-* [Exploring data](#exploring-data)
-* [Navigating the timeline](#navigating-the-timeline)
+-   [Launching the demo](#launching-the-demo)
+-   [The Viewer panels](#the-viewer-panels)
+-   [Exploring data](#exploring-data)
+-   [Navigating the timeline](#navigating-the-timeline)
 
 Here is a preview of the dataset that we will be working with:
 
 <picture>
-  <source media="(max-width: 480px)" srcset="https://static.rerun.io/viewer_walkthrough0_preview/d63e6774d94ff403d51355bacdfee9a3e7751dcf/480w.png">
-  <source media="(max-width: 768px)" srcset="https://static.rerun.io/viewer_walkthrough0_preview/d63e6774d94ff403d51355bacdfee9a3e7751dcf/768w.png">
-  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/viewer_walkthrough0_preview/d63e6774d94ff403d51355bacdfee9a3e7751dcf/1024w.png">
-  <img src="https://static.rerun.io/viewer_walkthrough0_preview/d63e6774d94ff403d51355bacdfee9a3e7751dcf/full.png" alt="viewer walkthrough dataset preview screenshot">
+  <img src="https://static.rerun.io/viewer_walkthrough_preview/eba55ea05deef665393cf42cc48d1236b4cc9692/full.png" alt="">
```

Comment:
as per note on slack dm: We really shouldn't have the `debug viewer` show on those screenshots. Either run a release build or use the `take a screenshot` feature from Rerun's command pallete for a "standardized" screenshot of the viewer which suppresses this banner

Reply:
will replace them!